beacon-chain/rpc/eth/beacon/handlers_pool_test.go

```diff
@@ -1763,8 +1726,7 @@ func TestGetAttesterSlashings(t *testing.T) {
 			ss, err := structs.AttesterSlashingsElectraToConsensus(slashings)
 			require.NoError(t, err)
 
-			require.DeepEqual(t, slashing1PostElectra, ss[0])
-			require.DeepEqual(t, slashing2PostElectra, ss[1])
```

Comment:
why remove the second one, i think this was testing whether the list was returning correctly

Reply:
because there can be only 1 Electra slashing in a block